zebra-consensus/src/chain.rs

```diff
@@ -79,33 +90,82 @@ where
         let mut state_service = self.state_service.clone();
         let max_checkpoint_height = self.max_checkpoint_height;
 
+        let height = block.coinbase_height();
+        // Report each 1000th block at info level
+        let info_log = matches!(height, Some(BlockHeight(height)) if (height % 1000 == 0));
+
+        if info_log {
+            tracing::info!(?height, "ChainVerifier received block");
+        } else {
+            tracing::debug!(?height, "ChainVerifier received block");
+        }
```

Comment:
Per https://github.com/ZcashFoundation/zebra/issues/695#issuecomment-662002171, I don't think we should do this kind of ad-hoc filter construction, it should be up to the caller to decide what filters they want to use, not source's job to filter on their behalf.

Reply:
If we change the way tracing is used here, I think it's possible to get this behavior back using tracing filters.

Reply:
There is a bunch of temporary diagnostic code in this PR. I'm not sure how to write tracing filters, but we can open another ticket to make it happen.

Let me know if you'd like me to drop this commit in the meantime.

Reply:
What do you think of d2b0ecc ? The thing that I was thinking of is (after adding a span) trying to do
```
set -x ZEBRAD_LOG "[block_verify{height=Some(.*000)}]=trace"
```
to match by regex on the height field.  But I can't actually get this to work, I think I got the syntax wrong.  So the suggestion I was thinking of may not actually be a workable one.

Reply:
`set -x ZEBRAD_LOG "[block_verify{height=Some(BlockHeight(.*000))}]=trace"` ?